src/CoWSwapEthFlow.sol

```diff
@@ -124,12 +124,24 @@ contract CoWSwapEthFlow is
     /// @inheritdoc ICoWSwapEthFlow
     function deleteOrders(EthFlowOrder.Data[] calldata orderArray) external {
         for (uint256 i = 0; i < orderArray.length; i++) {
-            deleteOrder(orderArray[i]);
+            _deleteOrder(orderArray[i], false);
         }
     }
 
     /// @inheritdoc ICoWSwapEthFlow
     function deleteOrder(EthFlowOrder.Data calldata order) public {
+        _deleteOrder(order, true);
+    }
+
+    /// @dev Marks an existing ETH-flow order as invalid and refunds the ETH that hasn't been traded yet.
+    /// Note that some parameters of the orders are ignored, as for example the order expiration date and the quote id.
```

Comment:
```suggestion
    /// @dev Performs the same tasks as `deleteOrder` (see documentation in `ICoWSwapEthFlow`) but also allows the
    /// caller to ignore the revert condition `NotAllowedToDeleteOrder`. Instead of reverting, it stops execution
    /// without causing any state change.
```